cellprofiler/modules/enhanceedges.py

```diff
@@ -55,79 +58,102 @@ def create_settings(self):
             What do you want to call the image with edges enhanced?''')
```

Comment:
Documentation strings for `self.image_name` and `self.output_image_name` can each be on one line.